youtubesearchpython/internal/streamurlfetcher.py

```diff
@@ -1,3 +1,5 @@
+import httpx
```

Comment:
I don't see this import being used anywhere.

Reply:
Oh, yeah. This is left from testing :sweat_smile: